ci/coverage_publish.sh

```diff
@@ -34,7 +34,7 @@ if [ "${TRAVIS_SECURE_ENV_VARS}" == "true" ]; then
   echo "aws_secret_access_key=${COVERAGE_AWS_SECRET_ACCESS_KEY}" >> ~/.aws/config
   echo "region=us-east-1" >> ~/.aws/config
 
-  aws s3 cp "${COVERAGE_DIR}" "s3://${S3_LOCATION}" --recursive --profile coverage --acl public-read --quiet
+  aws s3 cp "${COVERAGE_DIR}" "s3://${S3_LOCATION}" --recursive --profile coverage --acl public-read --quiet --sse
```

Comment:
for some context, enforcing encrypted writes can only be done at the bucket level (we have an OKR to flip this switch on >50% of buckets). So the question for you guys is: is _everything_ in the lyft-envoy bucket public, or should we eventually be enforcing encrypted writes? If the latter, we should move forward with this PR. It should have minimal to no impact on latency. We can test the visibility with a manual, encrypted upload before merging this PR as well, just to be sure.